### DISCUSSION TITLE
Maintenance Mode enabled

### DISCUSSION THREAD

**Discussion by monsterflo** [alexbelgium/hassio-addons#870] [Q&A]
Hello all,
I have a Nextcloud instance runniing on Home Assistant based on this distribution. Yesterday the maintenance mode suddenly enabled and I didn't find any solution after half a day of research.

Can I fix this problem with enviromental variables or a docker command?

Thanks in advance
Florian

**Comment by alexbelgium** [comment]
Hi, in theory when you start the add-on it automatically performs different maintenance activities and should disable it. If it doesn't, you can use my scripting system to execute the command to stop the maintenance 

1. With my scripting system (instructions : https://github.com/alexbelgium/hassio-addons/wiki/Add-ons-feature-:-customisation)
- Create a file named /config/addons_autoscripts/nextcloud_ocr.sh
- Write in the file content
```bash
#!/bin/bash
sudo -u abc -s /bin/bash -c "php /data/config/www/nextcloud/occ maintenance:mode --off"
rm /config/addons_autoscripts/nextcloud_ocr.sh
```
- Save file and restart addon

2. With Portainer (instructions : https://github.com/alexbelgium/hassio-addons/wiki/Portainer)
- Install the portainer addon
- Start a console session in your nextcloud container
- Run the command `sudo -u abc -s /bin/bash -c "php /data/config/www/nextcloud/occ maintenance:mode --off"`

**Reply by monsterflo** [reply]
Hello Alex,
thanks for the support. Id did half the trick. My Nextcloud is now stuck in "Update in process" and again I can't find a solution to fix that.

the updater.log has the following:

```
2023-06-17T04:27:45+0000 pB6hqEVIsv [info] updater cli is executed
2023-06-17T04:27:45+0000 pB6hqEVIsv [info] currentStep()
2023-06-17T04:27:45+0000 pB6hqEVIsv [info] Step 9 is in state "start".
2023-06-27T08:48:42+0000 zWvR4iTNWq [info] updater cli is executed
2023-06-27T08:48:42+0000 zWvR4iTNWq [info] currentStep()

```


Any ideas?

Thanks in advance
Florian

**Comment by alexbelgium** [comment]
Have you tried just restarting the addon, and have you removed your "/config/addons_autoscripts/nextcloud_ocr.sh" ?

You could try running manually `sudo -u abc -s /bin/bash -c "php /data/config/www/nextcloud/updater/updater.phar --no-interaction"`

**Comment by monsterflo** [comment]
Hello Alex,
I tried removed the .sh and restarted Home Assistant and the addon several times.

This is the result of your command:
```

oot@db21ed7f-nextcloud-ocr:/# sudo -u abc -s /bin/bash -c "php /data/config/www/nextcloud/updater/updater.phar --no-interaction"
Nextcloud Updater - version: v26.0.0beta2-9-g64e2e4c dirty


In Updater.php line 1066:
                                
  Can't decode .step JSON data  
                                

update [--no-backup] [--no-upgrade]
```

Is there any solution other than re-installing the addon from scratch?

Regards
Florian

**Comment by alexbelgium** [comment]
Oups I forgot you. It's better to create issues rather than discussion. You could try to delete the .step file (or rename it) located in your data folder
